devices/common_patches/merge_USART_CR1_DEATx_fields.yaml

```diff
@@ -0,0 +1,6 @@
+# Merge USART CR1 DEATx fields
+
+"USART*,LPUART*":
```

Comment:
This syntax doesn't do what you want, because the `USART*` peripheral can be processed before this one and then error when it can't find `DEAT*`. I think in general you don't want to have any peripheral block which is targeted by two different YAML blocks.

In this case can you just have a `USART*` and a `LPUART*` section separately, both with the same content? Either in the same file or (annoyingly) in different ones.

Reply:
(They'll have to be in different ones for devices that only have USART and no LPUART)

Reply:
I put them on separate files as some devices need USART only and other need both.